vpr/src/base/echo_files.cpp

```diff
@@ -158,9 +160,9 @@ void alloc_and_load_output_file_names(const std::string default_name) {
     char* name;
 
     if (outputFileNames == nullptr) {
-        outputFileNames = (char**)vtr::calloc((int)E_FILE_END_TOKEN, sizeof(char*));
+        outputFileNames = new char*[(int)E_FILE_END_TOKEN]();
 
-        name = (char*)vtr::malloc((strlen(default_name.c_str()) + 40) * sizeof(char));
+        name = new char[(strlen(default_name.c_str()) + 40)];
```

Comment:
This "40" magic number makes me nervous.
- Best: can we change this to C++ strings (no size issues anymore)?  If not too much work, that would be good.
- If not, change the 40 to a meaningful defined constant (have to read the code to see what it should be) and make it considerably bigger -- say 1024 -- so we never have an issue with it.